WalkingChallenge/SupportersViewController.swift

```diff
@@ -129,8 +130,10 @@ fileprivate class SponsorsDataSource: NSObject, UITableViewDataSource {
 fileprivate class SponsorsDelegate: NSObject, UITableViewDelegate {
   func tableView(_ tableView: UITableView,
                  heightForRowAt indexPath: IndexPath) -> CGFloat {
-    // FIXME(compnerd) calculate this somehow?
-    return 100.0
+    return UITableViewAutomaticDimension
+  }
```

Comment:
Would be nice to have a newline between the functions, and to wrap the arguments (80-col :-D)